js/Grid/Paging.md

```diff
@@ -57,7 +57,7 @@ $(function () {
 
 The following output is displayed as a result of the above code example.
 
-![](paging_images/paging_img2.png)
+![Pager with query string in JS Grid](paging_images/paging_img2.png)
```

Comment:
Use JavaScript instead of JS in all places

Reply:
Updated